matchms/exporting/save_as_msp.py

```diff
@@ -56,6 +59,9 @@ def save_as_msp(spectrums: List[Spectrum], filename: str,
                        filename.split(".")[-1])
     spectrums = _ensure_list(spectrums)
 
+    if style != "matchms":
+        list(map(lambda x: x.clone().update_keys(style), spectrums))

```

Comment:
```suggestion
        spectrums = [x.clone().update_keys(style) for x in spectrums]

```

Reply:
linter actually complains about not using generator expressions I assume.